packages/web-components/fast-foundation/src/radio-group/radio-group.ts

```diff
@@ -136,7 +136,7 @@ export class RadioGroup extends FASTElement {
         super.connectedCallback();
         this.direction = getDirection(this);
         this.setupRadioButtons();
-        this.parentToolbar = this.parentElement?.closest('[role="toolbar"]');
+        this.parentToolbar = this.closest<HTMLElement>('[role="toolbar"]');
         this.isInsideToolbar =
```

Comment:
I'm wondering if we should rename this to  something like "isTabStop" and make it a settable public api because there may be other situations where an author may want to do this, like say a radio group in a grid cell where authors want to follow "only one of the focusable elements contained by the grid is included in the page tab sequence." (https://www.w3.org/TR/wai-aria-practices/#grid).  We could then also consider consolidating detection in the parent (ie. toolbar) and have it check it's children for the presence of "isTabStop" and act accordingly.  Seems more scaleable.

Reply:
This situation is unique to a `radiogroup` inside of a `toolbar`: https://w3c.github.io/aria-practices/#for-radio-group-contained-in-a-toolbar

Reply:
I don't believe it is unique.  Data grids are typically a single tab stop yet can contain cells that contain multiple widgets which presumably could include a radio group, so in that situation an author would also want to suppress the behavior of radio groups adding child radio buttons to the tab queue.  There's even mention of it being valid for complex grid-cells possibly having their own contained tab queue like a modal dialog when a user has navigated "into" it.  Implementing this such that any parent, be it toolbar, grid-cell or something else can turn this behavior on or off seems more flexible to me.

Reply:
While there are similarities between keyboard navigation behaviors for composite widgets, this situation is specifically unique as defined in the [`radiogroup` section of the ARIA spec](https://w3c.github.io/aria-practices/#keyboard-interaction-15). For that reason, this property and its surrounding behaviors are only relevant when a `fast-radio-group` is inside of a `fast-toolbar`.

Reply:
If you look though the [grid section of the Aria spec](https://w3c.github.io/aria-practices/#grid) it specifically mentions a radio group as a valid child of a complex cell, and discusses a few approaches to keyboarding and how the tab key could behave in these scenarios.  So someone putting a radio group in a data grid cell could very likely want to take it out of the tab order.  This seems like the same functionality toolbar needs so why not a mechanism that accommodates both?

Reply:
@scomea can you open an issue to look at exploring and possibly adding the behavior based on a deeper investigation into something like a layout grid (which we don't currently have). My concern here is that we'll end up w/ some scope creep outside the toolbar itself. I'm all for exploring if we can update radio to leverage this in data grid as well, I just want to be careful to randomize as part of this PR and make it dependent on that change. Thoughts?

Reply:
Data grid currently supports complex cells which could have their own internal navigation queues.  Conceptually, I think that any component that add things to the tab queue automatically we'd probably want to be able to suppress that behavior (I'm thinking it will come up when someone wants to do a nested grid, for example). Also, I think it's a better design to have the parent suppress the behavior of the child (ie. toolbar suppresses the behavior in radio group) than the child try to determine the right behavior by detecting the parent (having to update the child for every new scenario isn't great).  I'll open an issue, but it seems like something we should standardize on.

Reply:
When I first looked over the code I also felt the child determining behavior weird. Components should be instructed on how to act rather than self-determining how to act in any given context.

I don't feel this should block merge though. This seems to be one of the first times this has come up (although I haven't deeply reviewed the other component code at all) and it's fine enough to land and able to be refined. As brought up with Datagrid, once more higher-order components are built these situations will be hit more. So these are good early sample cases to chomp on.